### DISCUSSION TITLE
Set of Basic Skills

### DISCUSSION THREAD

**Discussion by Odoamar** [fmasa/wfrp-master#100] [Ideas]
Unlike Advanced skills and Talents, basic skills are available for all characters/NPCs and sometimes it is not easy to track what statistic is related to them. 
I would like to see an additional button to add them all together rather than selecting all individually. I hope it will be easy to implement because some skills have the boolean flag for Advanced skills, right?  

This feature can save a lot of time during NPC creation.

**Comment by fmasa** [comment]
Yes, this should be fairly easy. I created the issue #101 and will tackle this in the near future :slightly_smiling_face:
